src/main/resources/application.properties

```diff
@@ -117,11 +118,13 @@ spring.application.name=ccd-data-store
 http.client.max.total=${HTTP_CLIENT_MAX_TOTAL:100}
 http.client.seconds.idle.connection=${HTTP_CLIENT_SECONDS_IDLE_CONNECTION:120}
 http.client.max.client_per_route=${HTTP_CLIENT_MAX_CLIENT_PER_ROUTE:20}
-http.client.validate.after.inactivity=${HTTP_CLIENT_VALIDATE_AFTER_INACTIVITY:0}
+http.client.validate.after.inactivity=${HTTP_CLIENT_VALIDATE_AFTER_INACTIVITY:100}
```

Comment:
During performance testing Fatih and I spotted an issue (org.apache.http.NoHttpResponseException) with server dropping connections under certain load and a solution on spring forums was suggested to add retry handler and also increase http.client.validate.after.inactivity property to 100 from intial 0 to help detect connections that have become stale (half-closed) while kept inactive in the pool.